include/vrv/options.h

```diff
@@ -602,6 +602,7 @@ class Options {
     OptionBool m_preserveAnalyticalMarkup;
     OptionBool m_removeIds;
     OptionBool m_shrinkToFit;
+    OptionBool m_centerStaccato;
```

Comment:
Please rename to staccatoCenter so that other possible future options on staccato are grouped together